dll/keyboard/kbdcr/kbdcr.c

```diff
@@ -41,7 +41,7 @@ ROSDATA USHORT scancode_to_vk[] = {
   /* - 0f - */
   /* First Letters Row */
   VK_TAB,       'Q',          'W',          'E',
-  'R',          'T',          'Y',          'U',
+  'R',          'Z',          'Y',          'U',
```

Comment:
Shouldn't it be
```suggestion
  'R',          'T',          'Z',          'U',
```
?

Reply:
Always I did something wrong; yeah I did mistake and I will fix it